sale_forecast/tests/test_sale_forecast.py

```diff
@@ -0,0 +1,316 @@
+            ('name', '=', 'Public Pricelist')])[0]
+
+        so_vals = {
+            'partner_id': self.partner_agrolite.id,
+            'pricelist_id': pricelist.id,
+            'order_line': [
+                (0, 0, {
+                    'name': self.productsf.name,
+                    'product_id': self.productsf.id,
+                    'product_uom_qty': 1.0,
+                    'product_uom': uom_id.id,
+                    'price_unit': 121.0
+                })
+            ]
+        }
+        so_vals2 = {
+            'partner_id': self.partner.id,
+            'pricelist_id': pricelist.id,
+            'order_line': [
+                (0, 0, {
+                    'name': self.productsf.name,
+                    'product_id': self.productsf.id,
+                    'product_uom_qty': 1.0,
+                    'product_uom': uom_id.id,
+                    'price_unit': 121.0
+                })
+            ]
+        }
+        so_vals3 = {
+            'partner_id': self.partner.id,
+            'pricelist_id': pricelist.id,
+            'order_line': [
+                (0, 0, {
+                    'name': self.productsf.name,
+                    'product_id': self.productsf.id,
+                    'product_uom_qty': 2.0,
+                    'product_uom': uom_id.id,
+                    'price_unit': 121.0
+                })
+            ]
+        }
+        so_vals4 = {
+            'partner_id': self.partner_agrolite.id,
+            'pricelist_id': pricelist.id,
+            'order_line': [
+                (0, 0, {
+                    'name': self.productsf.name,
+                    'product_id': self.productsf.id,
+                    'product_uom_qty': 2.0,
+                    'product_uom': uom_id.id,
+                    'price_unit': 121.0
+                })
+            ]
+        }
+        sf_vals = {
+            'name': 'Test 1',
+            'date_from': date.today() + relativedelta(years=2),
+            'date_to': date.today() + relativedelta(years=1),
+        }
+        with self.assertRaises(ValidationError):
+            self.sf_model.create(sf_vals)
+        sf_vals = {
+            'name': 'Test 1',
+            'date_from': date.today() + relativedelta(years=1),
+            'date_to': date.today() + relativedelta(years=2),
+        }
+        self.sf_model.create(sf_vals)
+
+        self.so_model.create(so_vals)
+        confirmed_so = self.so_model.create(so_vals2)
+        confirmed_so.action_confirm()
+        confirmed_so2 = self.so_model.create(so_vals3)
+        confirmed_so2.action_confirm()
+        confirmed_so3 = self.so_model.create(so_vals4)
+        confirmed_so3.action_confirm()
+
+        sf = self.sf_model.create(sf_vals)
+        context = {
+            "active_model": 'sale.forecast',
+            "active_ids": [sf.id],
+            "active_id": sf.id
+            }
+
+        load_sales_wizard = self.env['sale.forecast.load'].with_context(
```

Comment:
@carlosDomatix IMO Create a global variable `self.env['sale.forecast.load']` because it's use multi time